source/help/settings/account-settings.md

```diff
@@ -62,7 +62,9 @@ A notification sound plays for all Mattermost posts that would fire a desktop no
 Email notifications are sent for mentions and direct messages after you’ve been offline for more than 60 seconds or away from Mattermost for more than 5 minutes. Change the email where notifications are sent from **Account Settings > General > Email**.
 
 #### Mobile Push Notifications
-Push notifications are sent to your mobile device if you have the Android or iOS app installed. They are sent even if you are online, similar to desktop notifications. By default, push notifications are sent "For mentions and direct messages". If your System Administrator has not set up push notifications, this setting will be disabled.
+Push notifications can be sent to your mobile device if you have the Android or iOS app installed. When they are sent can be adjusted depending on the type of notification and your status. By default, push notifications are sent "For mentions and direct messages", and if your status is "Online, away or offline". If push notifications are sent "Never", the setting to trigger push notifications depending on your status is hidden. If your System Administrator has not set up push notifications, both settings will be disabled.
+
+Also, if you are actively viewing a channel (public, private, or direct message) on desktop or web, no push notifications will be sent for that channel. If a notification is viewed on desktop or web, any push notifications and app icon badges already sent will be cleared.
```

Comment:
I don't think push notifications are cleared for iOS yet (badge icon should be cleared though I think, you can confirm with Eric)